src/config/globalInfoConfig.ts

```diff
@@ -221,6 +221,12 @@ export class GlobalInfo extends ConfigFile<ConfigFile.Options, SfInfo> {
     aliases.forEach((alias) => this.unsetAlias(alias));
   }
 
+  public getUser(usernameOrAlias: string): string | null {
+    return (
```

Comment:
@RodEsp are the `()` necessary around the expression?

Reply:
VSCode auto-added them when it decided to put the expression on its own line.